sdk/python/packages/flet-core/src/flet_core/textfield.py

```diff
@@ -63,6 +65,13 @@ class TextCapitalization(Enum):
     SENTENCES = "sentences"
 
 
+@dataclasses.dataclass
+class InputFilter:
+    regex_string: str
```

Comment:
Looks good! If you want to "help" people you can add a few inherited from `InputFilter` classes with pre-defined regex, e.g. `NumbersOnlyInputFilter`, etc.

Reply:
Nice idea!
Will work on that later today.